UI/Web/src/app/carousel/_components/carousel-reel/carousel-reel.component.scss

```diff
@@ -30,7 +30,7 @@
 }
 
 ::ng-deep .swiper-slide {
-    width: auto !important;
+    width: 160px !important;
```

Comment:
I reverted this change when I merged. It breaks the Reviews:
![image](https://github.com/user-attachments/assets/38a76dc3-d1fc-4dfb-883f-b46aeb1636a1)

Reply:
Got it. I will look into fixing it other ways.